src/app/components/SearchResults/AdvancedSearchResults.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { initialSearchQuery, searchQueryPropTypes } from '../../stores/InitialState';
+import { getQueryString } from '../../search/query';
+
+const AdvancedSearchResults = ({ searchQuery, userQuery, router }) => {
+  // redirect to url with query params
+  const submit = (query) => {
+    const path = `/search?${getQueryString(query)}`;
+    router.push(path);
+  };
+  // update page in store and go to any page
```

Comment:
I really like having this functionality, but I don't think it was in the scope/ticket for this work?

The only reason that I raise this is that it allows the user to clear _all_ queries, resulting in the return of every work in the database. There isn't anything wrong with this, but it will put them in a situation that I hadn't anticipated and things such as paging will likely break if a user tries to take actions from this state.

I don't think this should be removed, since it is functionality we will almost certainly want in the future, but perhaps, the buttons could be disabled for now?

Reply:
I thought this is what is expected from this kind of tags, otherwise there is no way to remove advanced search terms

Reply:
Anyway now you can get to full results, just search in advanced search for nothing with a language, then remove the language.

Reply:
With the discussion we just had I think it makes sense to make them static. I agree that its the generally expected behavior, but it just intersects with too many other decisions about how we return results to be activated right now.